### ISSUE TITLE
Windows binary included in readpe with no source

### ISSUE THREAD

**Issue by davidpolverari** [mentebinaria/readpe#210]
**Describe the bug**
While I was packaging the latest readpe version for Debian, I realized the file [support_files/samples/hh6d.golden.exe](https://github.com/mentebinaria/readpe/commit/9e4b950dbac0eedf28c3b6b3ff00db90aa613436#diff-6dd08c7626f70dd2fac1063f3f9b5bb96a2f699db31bbe0840b6a456d64c799a), which was added as a sample "hello world" binary for tests, comes with no corresponding source, nor a build system to generate it.

From the Debian standpoint, this is a violation of its free software guidelines. I could repackage upstream sources without the binary to comply with DFSG, but this would not be the best solution, as tests would not be run.

Besides, from a little bit of research, it seems that the binary comes from the [pts-tinype](https://github.com/pts/pts-tinype) repo. By perusing the files over there, I noticed there was no license in any of the files. This makes its inclusion even in readpe somewhat concerning from a legal standpoint.

**To Reproduce**
Please provide us with:
- pev version 0.83 (https://github.com/mentebinaria/readpe/commit/7f3c13649959747ec57db172da355038c3521e1e)
- OS version: Debian GNU/Linux 12

**Comment by GoGoOtaku** [comment]
Thank you for bringing this to my notice. I take this very seriously.

When I approved the merge request I already had a bad feeling but I had made sure to find out where the file was from and I thought I also checked the license.
Turns out I should have trusted myself more and rejected it.

I removed the file from the repository and it's history and remade the release.
(Turns out github really doesn't like recreating tar.gz files hence why I had to do it a couple times - sorry for the mail spam whoever follows this repository)
Now the release obviously is missing the test file but I rather have a release with broken tests than have this executable in here. I will create a small executable myself and add it and it's source code to the repository.

Once again: This is very serious and I really dropped the ball here. I'm sorry.
